.github/workflows/ci.yml

```
@@ -0,0 +1,43 @@
+on:
+  - push
```

```suggestion
  push:
    branches:
      - master
```

This will avoid running each test twice on Pull Requests (CI will run for push only when pushes go to `master`)

---

good idea!